.github/workflows/pr_ci.yml

```diff
@@ -15,17 +15,25 @@ jobs:
     name: Build & Test
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-node@v2
+# Node.JS 16 is included by default
+#    - uses: actions/setup-node@v2
+#      with:
+#        node-version: "15"
+# NET 6 is included by default
+#    - name: Setup .NET 6.0
+#      uses: actions/setup-dotnet@v1
+#      with:
+#        dotnet-version: 6.0.x
+#        include-prerelease: true
```

Comment:
You can remove the comments. I think it is well explained in the PR.

Reply:
I've removed the commented-out lines